File: app/src/main/java/com/sample/demo/dao/Record.java
package com.sample.demo.dao;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Record {

    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("age")
    @Expose
    public String age;
    @SerializedName("gender")
    @Expose
    public String gender;
    @SerializedName("address")
    @Expose
    public String address;

}

File: app/src/main/java/com/shivam/demo/dao/Record.kt
package com.shivam.demo.dao

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Record : Serializable {

    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("age")
    @Expose
    var age: String? = null
    @SerializedName("gender")
    @Expose
    var gender: String? = null
    @SerializedName("address")
    @Expose
    var address: String? = null

}